README.md

```diff
@@ -455,33 +446,36 @@ Then, we need to start Orchestrate again:
 npm run down && npm run up
 ```
 
-> IMPORTANT: In case of running in a production environment, please ensure you generate your own
-> certificate and private keys, using for instance `openssl`.
+#### Using self-generated certificates
+
+For the sake of simplicity Orchestrate team has provided a dev certificate
+to encode and decode the generated JWT. In case you want to use your own certificates, generated for instance with `openssl`, you need to fill up the following two ENV vars:
+
+```
+AUTH_JWT_CERTIFICATE=
+AUTH_JWT_PRIVATE_KEY=
+```
 
 ### Generate JWT token
 
-To generate a valid JWT we need to run the following command, where `TENANT_ID=` indicate the tenant id
-included in our token
+To generate a valid JWT we need to run the following command, where `TENANT_ID=` indicate the tenant id included in our token.
 
 ```
 TENANT_ID=foo npm run generate-jwt
 ```
 
-### How to test it
+By default the _expiration time is 1h_, in case you want to modify you can use the ENV variable _EXPIRATION_. For instance, to make it 5 hours you execute:
 
-Now we need to append the obtained token into every request we do to Orchestrate, as follow:
-
-```js
-  const requestId = await producer.sendTransaction({
-    chain: process.env.CHAIN!,
-    contractName: 'Counter',
-    methodSignature: 'increment(uint256)',
-    args: [1],
-    authToken: 'Bearer {{GENERATED_TOKEN}}',
-    to: process.env.TO_ACCOUNT,
-    from: process.env.FROM_ACCOUNT!
-  })
-}
 ```
+EXPIRATION=5h TENANT_ID=foo npm run generate-jwt
+```
+
+![Output of the generate-jwt command](static/generate-jwt.png)
+
+> **Note:** decode the generate token using _[https://jwt.io/](https://jwt.io/)_ and verify the correct tenant is included within selected namespace.
```

Comment:
the generated token. 
Within the selected namespace

Reply:
Oki